### PR TITLE
Single article

### PR DESCRIPTION
**Pull request by TypeError92**

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    
+            expect(article.article_id).toEqual(expect.any(Number));
+            expect(article.title).toEqual(expect.any(String));
+            expect(article.topic).toEqual(expect.any(String));
+            expect(article.author).toEqual(expect.any(String));
+            expect(article.created_at).toEqual(expect.any(String));
+            expect(article.votes).toEqual(expect.any(Number));
+            expect(article.article_img_url).toEqual(expect.any(String));
+            expect(article.comment_count).toEqual(expect.any(Number));
+            });
```
Comment (johnhutcheon) [inline __tests__/api.test.js:27]: Although what you've done here is technically correct, we don't actually need any testing! This is just a duplicate of what we did in the back end for this endpoint. We already know that when we go to the endpoint it's going to give us these keys as it's previously been tested, so no need to do this by checking the axios functions too. You don't need to worry about any testing like this for this project 👍

File: src/api.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import axios from 'axios'
+const api = axios.create({baseURL: 'https://nc-news-yxyq.onrender.com/api'})
+
+export function fetchArticles(){
+    return api.get('/articles')
+}
+
+export function fetchArticle(article_id){
+    return api.get(`articles/${article_id}`)
```
Comment (johnhutcheon) [inline src/api.js:9]: Thanks for implementing the feedback from the previous PR! 🙌

**Review by johnhutcheon — COMMENTED** [review]
Nice work Simon! Just left a bit of feedback in my comments, no need for testing here! I also assume you'll be adding it later on, but might just want to add in a link back to the Home page/all articles from our single article page. 👍
